### ISSUE TITLE
Temporary failure resolving 'http.kali.org' prevents install of metapackages

### ISSUE THREAD

**Issue by reybango** [offensive-security/kali-nethunter#937]
### Device:

Nexus 5

### OS version (KitKat/Lollipop/Marshmallow/Nougat):

Marshmallow Android 6.0.1

### Built from repo (date and build command) or downloaded from website (links):

https://images.offensive-security.com/nethunter-release/nethunter-hammerhead-marshmallow-3.0.zip

### Output of `cat /proc/version` in adb shell or NetHunter Terminal:

Linux version 3.4.0-gb7558f2 (root@kali) (gcc version 4.7 (GCC) #2 SMP PREEMPT Wed Oct 7 02:16:12 UTC 2015

## Issue:

Error message when trying to install new metapackages or apt-get update. 

**Entering apt-get update produces the following:**

Err:1 http://http.kali.org/kali kali-rolling InRelease
Temporary failure resolving 'http.kali.org'
Reading package lists... Done
W: Failed to fetch http://http.kali.org/kali/dists/kali-rolling/InRelease  Temporary failure resolving 'http.kali.org'
W: Some index files failed to download. They have been ignored, or old ones used instead.

**Using the Nethunter app to install the wireless metapackage produces the following:**

Err:1 http://http.kali.org/kali/dists/kali-rolling/main armhf python-minimal armhf 2.7.13-2
 (Followed by 268 similar error messages saying the same for different packages)

**Then:**

E: Failed to fetch http://http.kali.org/kali/pool/main/python-defaults/python-minimal_2.7.13-2armhf.deb  Temporary failure resolving 'http.kali.org' 
 (Followed by lots similar error messages saying the same for different packages)

**The last message is:**
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing (which I tried)

I have the latest APK installed and interestingly enough, just before I tried to install this package, it looked like I was successful in installing the web metapackage.

'/etc/apt/sources.list' is set to this:
"deb http://http.kali.org/kali kali-rolling main contrib non-free"

It was originally pointing to:
"deb http://http.kali.org/kali sana main non-free contrib 
deb http://http.kali.org/kali sana/updates main contrib non-free"

This seems to be the exact problem that was mentioned here https://github.com/offensive-security/kali-nethunter/issues/913

**Comment by reybango** [comment]
For reference I found this bug that's similar in the Kali bugtrack https://bugs.kali.org/view.php?id=3012

**Comment by reybango** [comment]
More info, doing what was mentioned here:

https://github.com/offensive-security/kali-nethunter/issues/913#issuecomment-312156669

and here:

https://forums.kali.org/showthread.php?33399-Cannot-apt-get-update-Err-1-http-http-kali-org-kali-kali-rolling-InRelease&p=65835&viewfull=1#post65835

resolved the issue. I changed the UID for _apt to 0 like was mentioned and all of sudden everything started working. I was able to apt-get and install metapackages via Nethunter.

So it seems that this bug still persists in recent builds: https://bugs.kali.org/view.php?id=3012
